### ISSUE TITLE
AWS Serverless cannot find YAML, default project

### ISSUE THREAD

**Issue by JUNGQUI** [aws/aws-toolkit-jetbrains#856]
**Describe the bug**
I make a simple python project by AWS Serverless Application and setting SAM Template AWS SAM Hello World, but after make a project, IDE occur error (exactly, I think plugins error) and error says java.lang.AssertionError: Found 0 YAML files but only expected 1.
and after than project well made but setting run/debug configuration AWS lambda, Error occur again and say cannot find handler 'app.hello_word' in project.

As we know that handler is default handler when choose template aws hello.

I don't understand why this problem is occured.

**To reproduce**

1. Open pycharm and create new project
2. Choose AWS Serverless Application, setting SAM Template AWS SAM Hello World, setting interpreter and location.
3. Create, than you can see IDE error.
4. But project is looks like good to go. than edit Run/Debug Configurations, add AWS Lambda.
5. Filling in like image.

**Expected behavior**

Just start.

**Screenshots**
![image](https://user-images.githubusercontent.com/39726558/55145067-62b46300-5185-11e9-878a-e276482f764c.png)

**Your Environment**
 - OS: windows 10
 - JetBrains' Product: pycharm
 - JetBrains' Product Version: 2018.3.5
 - Toolkit Version: 1.2
 - SAM CLI Version: 0.14.2
 - JVM/Python Version: 3.7

**Additional context**

Maybe this situation because of my bad understanding on plug-in. but i want to default running at least just start.

**Comment by gliptak** [comment]
I also see an error while creating Python Hello World. Here is the stacktrace shown:

```
java.lang.AssertionError: Found 0 YAML files but only expected 1
	at software.aws.toolkits.jetbrains.services.lambda.sam.SamCommon$Companion.getTemplateFromDirectory(SamCommon.kt:143)
	at software.aws.toolkits.jetbrains.services.lambda.sam.SamCommon$Companion.setSourceRoots(SamCommon.kt:169)
	at software.aws.toolkits.jetbrains.ui.wizard.SamProjectBuilder.setupRootModel(SamProjectGeneratorIntelliJShims.kt:63)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.setupModule(ModuleBuilder.java:251)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.createModule(ModuleBuilder.java:244)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.createAndCommitIfNeeded(ModuleBuilder.java:283)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.lambda$commitModule$3(ModuleBuilder.java:326)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteActionWithClass(ApplicationImpl.java:1020)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:1070)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.commitModule(ModuleBuilder.java:325)
	at com.intellij.ide.util.projectWizard.ModuleBuilder.commit(ModuleBuilder.java:311)
	at com.intellij.ide.impl.NewProjectUtil.doCreate(NewProjectUtil.java:138)
	at com.intellij.ide.impl.NewProjectUtil.createFromWizard(NewProjectUtil.java:60)
	at com.intellij.ide.impl.NewProjectUtil.createNewProject(NewProjectUtil.java:54)
	at com.intellij.ide.actions.NewProjectAction.lambda$actionPerformed$0(NewProjectAction.java:26)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:747)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:696)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by abrooksv** [comment]
Thank you for the stack trace, it will make it much easier to fix.

The code that fails is when it is marking the folders as source roots, which is required for the handler resolution. You can manually set the folder as a source root as a workaround by right clicking on the `hello_world` folder and going to `Mark Directory As -> Source Root`

**Comment by gliptak** [comment]
I do not see ```Mark Directory As -> Source Root``` menu option, so there might some other problems with my project ...

**Comment by abrooksv** [comment]
It looks like this for me:
![Screen Shot 2019-03-28 at 1 48 41 PM](https://user-images.githubusercontent.com/8992246/55191775-42be7880-5160-11e9-8e2c-55b433c2eafb.png)

**Comment by JUNGQUI** [comment]
Thanks to you @abrooksv ! your way make me smooth coding!

**Comment by gliptak** [comment]
I saw a Python crash dialog this morning and now the menu shows up for me. Thank you for the pointer and the quick patch @abrooksv

**Comment by zhangzhx** [comment]
This fix is released in the [version 1.3](https://plugins.jetbrains.com/plugin/11349-aws-toolkit/update/61424). Please let us know if you have any more issues from trying the new version.
